ChaosMod/Effects/db/Player/PlayerBirthday.cpp

```diff
@@ -0,0 +1,110 @@
+/*
+	Effect By OnlyRealNubs, made in celebration of my birthday
+*/
+
+#include "stdafx.h"
+
+//props, peds, dicts and other vars
+std::string ptfxDict = "core";
+std::string ptfxName = "ent_dst_gen_gobstop";
+std::string pedModel = "s_m_y_clown_01";
+std::string cakeModel = "v_res_cakedome";
+std::string audDict = "DLC_TG_Running_Back_Sounds";
+std::string audName = "Cheers";
+
+int soundId = 0;
+
+static void LoadStuff()
+{
+	//load models
+	std::vector<std::string> modelsToLoad = {pedModel, cakeModel};
+	for (int i = 0; i < modelsToLoad.size(); i++)
+	{
+		while (!HAS_MODEL_LOADED(GET_HASH_KEY(modelsToLoad[i].c_str())))
+		{
+			REQUEST_MODEL(GET_HASH_KEY(modelsToLoad[i].c_str()));
+			WAIT(1);
+		}
+	}
+
+	//load ptfx
+	while (!HAS_NAMED_PTFX_ASSET_LOADED(ptfxDict.c_str()))
+	{
+		REQUEST_NAMED_PTFX_ASSET(ptfxDict.c_str());
+		WAIT(1);
+	}
+}
+
+static void UnloadStuff()
+{
+	//unload models
+	std::vector<std::string> modelsToLoad = {pedModel, cakeModel};
+	for (int i = 0; i < modelsToLoad.size(); i++)
+	{
+		if (!IS_MODEL_VALID(GET_HASH_KEY(modelsToLoad[i].c_str()))) continue;
+		while (HAS_MODEL_LOADED(GET_HASH_KEY(modelsToLoad[i].c_str())))
+		{
+			SET_MODEL_AS_NO_LONGER_NEEDED(GET_HASH_KEY(modelsToLoad[i].c_str()));
+			WAIT(1);
+		}
+	}
+
+	//unload ptfx
+	while (HAS_NAMED_PTFX_ASSET_LOADED(ptfxDict.c_str()))
+	{
+		REMOVE_NAMED_PTFX_ASSET(ptfxDict.c_str());
+		WAIT(1);
+	}
+}
+
+static void OnStart()
+{
+	LoadStuff();
+
+	Player player = PLAYER_ID();
+	Ped playerPed = PLAYER_PED_ID();
+	Vehicle plrVeh = 0;
```

Comment:
No need to declare it up there, you can declsre it at the correct position. (I.e. this at  line 80)